knora/dsplib/utils/onto_create_ontology.py

```diff
@@ -301,151 +303,189 @@ def create_ontology(input_file: str,
+                                                                          comment=res_comment).create(
+                        last_modification_date)
+                except BaseError as err:
+                    print(
+                        f"ERROR while trying to create resource class {res_name}. The error message was {err.message}")
+                except Exception as exception:
+                    print(
+                        f"ERROR while trying to create resource class {res_name}. The error message was {exception}")
+                new_res_classes[new_res_class.id] = new_res_class
+                new_ontology.lastModificationDate = last_modification_date
+
+                if verbose:
+                    print("Created resource class:")
+                    new_res_class.print()
 
         # create the property classes
-        propclasses = ontology["properties"]
-        newpropclasses: Dict[str, ResourceClass] = {}
-        for propclass in propclasses:
-            propname = propclass.get("name")
-            proplabel = LangString(propclass.get("labels"))
-            # get the super-property/ies if defined. Valid forms are:
-            #   - "prefix:superproperty" : fully qualified name of property in another ontology. The prefix has to
-            #     be defined in the prefixes part.
-            #   - "superproperty" : Use of super-property defined in the knora-api ontology
-            #  if omitted, "knora-api:hasValue" is assumed
-            if propclass.get("super") is not None:
-                super_props = list(map(lambda a: a if ':' in a else "knora-api:" + a, propclass["super"]))
-            else:
-                super_props = ["knora-api:hasValue"]
-            # get the "object" if defined. Valid forms are:
-            #  - "prefix:object_name" : fully qualified object. The prefix has to be defined in the prefixes part.
-            #  - ":object_name" : The object is defined in the current ontology.
-            #  - "object_name" : The object is defined in "knora-api"
-            if propclass.get("object") is not None:
-                tmp = propclass["object"].split(':')
-                if len(tmp) > 1:
-                    if tmp[0]:
-                        object = propclass["object"]  # fully qualified name
+        prop_classes = ontology.get("properties")
+        if prop_classes:
+            for prop_class in prop_classes:
+                prop_name = prop_class.get("name")
+                prop_label = LangString(prop_class.get("labels"))
+                '''
+                get the super-property/ies if defined, valid forms are:
+                  - "prefix:super-property" : fully qualified name of property in another ontology. The prefix has to be 
+                    defined in the prefixes part.
+                  - "super-property" : super-property defined in the knora-api ontology
+                  - if omitted, "knora-api:hasValue" is assumed
+                '''
```

Comment:
IMO, tripple quotes should be only for "real" docstrings, not for regular comments amidst the code